### PR TITLE
Resize Hardtack into Hardtack Crackers

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

#### Purpose of change
As part of #74874 this PR will increase the size of hardtack made into appropriately sized crackers.  7.6 cm x 7.6 cm x 1.26 cm.  
Also includes recipes to make bite sized chunks of hardtack from crackers which age faster considering the difference between area and volume in the two sizes.  

#### Describe the solution
Adds recipes and adds an item.

### CODE REVIEW

File: data/json/recipes/recipe_food.json [hunk 1]
```diff
@@ -4420,11 +4420,24 @@
...
     ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "hardtack",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_BREAD",
+    "skill_used": "cooking",
+    "difficulty": 0,
```
Comment (Maleclypse) [inline data/json/recipes/recipe_food.json:4435]:
```suggestion
```

File: data/json/recipes/recipe_food.json [hunk 2]
```diff
@@ -4420,11 +4420,24 @@
...
+    "charges": 6,
+    "time": "2 m",
+    "autolearn": true,
+    "components": [ [ "hardtack_cracker", 1 ] ]
```
Comment (Maleclypse) [inline data/json/recipes/recipe_food.json:4439]:
```suggestion
    "components": [ "hardtack_cracker", 1 ]
```

File: data/json/recipes/recipe_food.json [hunk 3]
```diff
@@ -4420,11 +4420,23 @@
...
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "hardtack",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_BREAD",
+    "skill_used": "cooking",
+    "charges": 6,
+    "time": "2 m",
+    "autolearn": true,
+    "components": [ "hardtack_cracker", 1 ]
```
Comment (Maleclypse) [inline data/json/recipes/recipe_food.json:4438]:
```suggestion
    "components": "hardtack_cracker", 1
```

File: data/json/recipes/recipe_food.json [hunk 4]
```diff
@@ -4420,11 +4420,23 @@
...
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "hardtack",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_BREAD",
+    "skill_used": "cooking",
+    "charges": 6,
+    "time": "2 m",
+    "autolearn": true,
+    "components": "hardtack_cracker", 1
```
Comment (Maleclypse) [inline data/json/recipes/recipe_food.json:4438]:
```suggestion
    "components": [ [ [ "hardtack_cracker", 1 ] ] ]
```

File: data/json/items/comestibles/bread.json [hunk 5]
```diff
@@ -443,6 +443,26 @@
...
+    "id": "hardtack_cracker",
+    "name": { "str_sp": "hardtack cracker" },
+    "weight": "480 g",
+    "color": "white",
+    "spoils_in": "360 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -12,
+    "calories": 280,
+    "description": "A dry, virtually tasteless bread cracker capable of remaining edible without spoilage for vast lengths of time.  So hard it is inedible by itself. Usually broken into smaller pieces for consumption.",
```
Comment (Maleclypse) [inline data/json/items/comestibles/bread.json:457]:
```suggestion
    "description": "A dry, virtually tasteless bread cracker capable of remaining edible without spoilage for vast lengths of time.  So hard it is inedible by itself.  Usually broken into smaller pieces for consumption.",
```

**Comment by Terrorforge** [comment]
A piece of hardtack this size should only weight about 70 grams. [Here](https://theprepared.com/survival-skills/guides/hardtack/)'s an article where they weighed a 3x3-inch piece of hardtack at 2.5 ounces. 

This has been wrong for a long time. As discussed in #61312 many bread products have wacky density, largely due to past misuse of the stack_size trait. Fixing that is of course out of scope for this PR, but for reference [here's](https://modernistcuisine.com/mc/bread-is-lighter-than-whipped-cream/) an article discussing the density of bread more generally; at around 1 gram per cubic centimetre, hardtack is among the densest.

File: data/json/items/comestibles/bread.json [hunk 6]
```diff
@@ -443,6 +443,26 @@
     "fun": 1,
     "vitamins": [ [ "calcium", 1 ], [ "iron", 1 ], [ "wheat_allergen", 1 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "hardtack_cracker",
+    "name": { "str_sp": "hardtack cracker" },
+    "weight": "480 g",
```
Comment (Maleclypse) [inline data/json/items/comestibles/bread.json:450]:
```suggestion
    "weight": "72 g",
```

**Comment by Maleclypse** [comment]
> A piece of hardtack this size should only weigh 71 grams. [Here](https://theprepared.com/survival-skills/guides/hardtack/)'s an article where they weighed a 3x3-inch piece of hardtack at 2.5 ounces.
> 
> This has been wrong for a long time. As discussed in #61312 many bread products have wacky density, largely due to past misuse of the stack_size trait. Fixing that is of course out of scope for this PR, but for reference [here's](https://modernistcuisine.com/mc/bread-is-lighter-than-whipped-cream/) an article discussing the density of bread more generally; at around 1 gram per cubic centimetre, hardtack is among the densest.

Good catch! Thanks!

**Pull request merged**